src/main/app/forms/models/partyDetails.ts

```diff
@@ -23,21 +32,23 @@ export class PartyDetails implements Serializable<PartyDetails>, CompletableTask
   correspondenceAddress?: CorrespondenceAddress
 
   constructor (
+    name: string = undefined,
```

Comment:
a value is undefined by default,
also name is on party not partyDetails...
why the move?